chalice/package.py

```diff
@@ -585,8 +585,6 @@ def generate(self, resources):
             },
             'provider': {
                 'aws': {'version': '> 2.0.0'},
-                'local': {'version': '> 1.0'},
-                'template': {'version': '> 2.0'},
```

Comment:
Hmm. We wanted to keep the `template` version floor constraint right?

Reply:
Also on a related topic @jamesls wanted to circle back about having a major version ceiling for these providers. He added this comment: https://github.com/aws/chalice/pull/1129#discussion_r310848785 and not sure if you saw it?

Reply:
updated with ceiling

Reply:
Awesome. Thanks!